src/h5web/visualizations/shared/VisCanvas.tsx

```diff
@@ -4,20 +4,25 @@ import { useMeasure } from 'react-use';
 import styles from './VisCanvas.module.css';
 import { AxisOffsets, AxisDomains } from './models';
 import { computeVisSize } from './utils';
-import AxisGrid from './AxisGrid';
+import AxisSystem from './AxisSystem';
 
 interface Props {
   axisOffsets: AxisOffsets;
   axisDomains?: AxisDomains;
   aspectRatio?: number;
+  showGrid?: boolean;
   children: ReactNode;
 }
 
 function VisCanvas(props: Props): JSX.Element {
-  const { axisDomains, axisOffsets, aspectRatio, children } = props;
+  const { axisDomains, axisOffsets, aspectRatio, children, showGrid } = props;
   const [visAreaRef, visAreaSize] = useMeasure();
+  const availableSize = {
+    width: visAreaSize.width - axisOffsets.left,
+    height: visAreaSize.height - axisOffsets.bottom,
+  };
 
-  const visSize = computeVisSize(visAreaSize, axisOffsets, aspectRatio);
+  const visSize = computeVisSize(availableSize, aspectRatio);
```

Comment:
Unrelated to the grid but I feel it is clearer when the computation of `availableSize` exists in the same scope of the padding addition.

Reply:
very true 👍